pkg/storage/ipfs_apicopy/storage.go

```diff
@@ -106,8 +106,12 @@ func (dockerIPFS *StorageProvider) PrepareStorage(ctx context.Context, storageSp
 
 //nolint:lll // Exception to the long rule
 func (dockerIPFS *StorageProvider) CleanupStorage(ctx context.Context, storageSpec model.StorageSpec, volume storage.StorageVolume) error {
-	return system.RunCommand("sudo", []string{
-		"rm", "-rf", fmt.Sprintf("%s/%s", dockerIPFS.LocalDir, storageSpec.Cid),
+	// TODO make sure sudo is not needed here
```

Comment:
@binocarlos Thoughts RE: using `sudo` here? It kinda breaks running tests locally because it prompts the user for sudo pwd. And overall it should not be needed because this func removes a temp-dir. I dropped it and all tests pass anyway :)

Reply:
WFM - feels well constrained to just test cleanup.

Reply:
👍